contracts/liquid-collateral/documentation.md

```diff
@@ -0,0 +1,85 @@
+# Contract storage
+
+## State
+- project_owner: address which is optionally set during initialization. If set, only this address will have permission rights to execute 'create_position' msg.
```

Comment:
why not just have these in the code itself? I am not sure this part of the doc does much